packages/aragon-wrapper/src/index.js

```diff
@@ -376,8 +377,51 @@ export default class Aragon {
       ))
     )
 
+    // SetApp events are emitted when apps are installed and upgraded
+    // These may modify the implementation addresses of the proxies (modifying their behaviour), so
+    // we invalidate any caching we've done
+    const updatedApps$ = this.kernelProxy
+      // Override events subscription with empty options to subscribe from latest block
+      .events('SetApp', {})
```

Comment:
If `fromBlock` is not sent on calls to `events`, does it only emit events from the latest block? 

It's not documented in https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#id34

Reply:
Yes, this is more or less buried in the code. It assumes latest if you don't include it or set it to `0`.

See [latest from master](https://github.com/ethereum/web3.js/blob/cd32de24bf/packages/web3-core-subscriptions/src/subscriptions/eth/LogSubscription.js#L53), and [the version we use (v1.0.0-beta.34)](https://github.com/ethereum/web3.js/blob/fcf6997d15/packages/web3-core-subscriptions/src/subscription.js#L222).

Made an issue for it: https://github.com/ethereum/web3.js/issues/2668.